specificatie/BRK-Bevragen/openapi.yaml

```diff
@@ -1280,6 +1283,24 @@ components:
       - properties:
           actueleStatutaireNaam:
             type: string
+    IngeschrevenNatuurlijkPersoonCompact:
+      type: object
+      description: "Natuurlijk persoon ingeschreven in de Basisregistratie Personen (BRP) met een burgerservicenummer"
+      properties:
+        burgerservicenummer:
+          type: string
+        naam:
```

Comment:
Willen we de naam niet gelijk maken met de naam (geslachtsnaam of aanschrijfwijze) die het voorstelt in de BRP?

Reply:
Aanschrijfwijze lijkt me zeker geen goed idee. Zou het Kadaster dan die logica moeten implementeren ? Vraag is even of hier inderdaad alleen de geslachtsnaam wordt gepresenteerd of uit convenience-overwegingen een samengestelde naam. Dit is trouwens sowieso een convenience-gegeven omdat de naam uit de BRP hoort te komen. 
Ik maak een issue aan.